src/components/Search/Search.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Search = ({ query, getQuery }) => (
+  <div className="box">
+    <div className="field">
+      <label htmlFor="search-query" className="label">
+        Search movie
+      </label>
+
+      <div className="control">
+        <input
+          type="text"
+          value={query}
+          id="search-query"
+          className="input"
+          placeholder="Type search word"
+          onChange={getQuery}
```

Comment:
```suggestion
          onChange={setQuery}
```
ты делаешь `setState`, так что это скорее `setQuery`